src/System.Management.Automation/utils/StructuredTraceSource.cs

```diff
@@ -1046,25 +1176,44 @@ internal void WriteLine(object arg)
         /// </param>
         /// <param name="classFormatter">
         /// This is the trace class formatter. For instance,
-        /// TraceError has a formatter like "ERROR: {0}"
+        /// TraceError has a formatter like "ERROR: {0}".
         /// </param>
         /// <param name="format">
-        /// Additional format string
+        /// Additional format string.
         /// </param>
-        /// <param name="args">
-        /// Arguments for the additional format string
+        /// <param name="arg1">
+        /// Argument for the additional format string.
+        /// </param>
+        /// <param name="arg2">
+        /// Argument for the additional format string.
+        /// </param>
+        /// <param name="arg3">
+        /// Argument for the additional format string.
+        /// </param>
+        /// <param name="arg4">
+        /// Argument for the additional format string.
+        /// </param>
+        /// <param name="arg5">
+        /// Argument for the additional format string.
+        /// </param>
+        /// <param name="arg6">
+        /// Argument for the additional format string.
         /// </param>
         private void FormatOutputLine(
             PSTraceSourceOptions flag,
             string classFormatter,
             string format,
-            params object[] args)
```

Comment:
Why removing the `param object[] args` overload? it's still used in this file ...